src/DIPOLE/pair_lj_cut_dipole_cut.cpp

```diff
@@ -90,6 +90,8 @@ void PairLJCutDipoleCut::compute(int eflag, int vflag)
   numneigh = list->numneigh;
   firstneigh = list->firstneigh;
 
+  int maxsize = 10;
```

Comment:
This change seems unrelated to the AMOEBA package.

Reply:
Right, this was a mistake. I removed these unwanted changes, reverting to the version of this file from develop.